.github/workflows/orca.yml

```diff
@@ -0,0 +1,90 @@
+      # Hardcode path since GITHUB_WORKSPACE can't be used here.
+      # @see https://github.community/t/how-to-use-env-context/16975/9
+      ORCA_SUT_DIR: /home/runner/work/orca/example
+      ORCA_SELF_TEST_COVERAGE_CLOVER: $HOME/build/logs/clover-self.xml
+      ORCA_JOB: ${{ matrix.orca-job }}
+
+    strategy:
+      matrix:
+        orca-job:
+          - STATIC_CODE_ANALYSIS
+          - INTEGRATED_TEST_ON_OLDEST_SUPPORTED
+          - INTEGRATED_TEST_ON_LATEST_LTS
+          - INTEGRATED_TEST_ON_PREVIOUS_MINOR
+          - INTEGRATED_UPGRADE_TEST_FROM_PREVIOUS_MINOR
+          - ISOLATED_TEST_ON_CURRENT
+          - INTEGRATED_TEST_ON_CURRENT
+          - INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR
+          - ISOLATED_TEST_ON_CURRENT_DEV
+          - INTEGRATED_TEST_ON_CURRENT_DEV
+          - STRICT_DEPRECATED_CODE_SCAN
+          - ISOLATED_TEST_ON_NEXT_MINOR
+          - INTEGRATED_TEST_ON_NEXT_MINOR
+          - ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_BETA_OR_LATER
+        php-version: ["7.3"]
+        allow-failure: [false]
+        include:
+          - orca-job: ISOLATED_TEST_ON_CURRENT
+            php-version: "8.0"
+            allow-failure: false
+          - orca-job: INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: LOOSE_DEPRECATED_CODE_SCAN
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: DEPRECATED_CODE_SCAN_W_CONTRIB
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: ISOLATED_TEST_ON_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: INTEGRATED_TEST_ON_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          coverage: xdebug2
+      - name: Before install
+        run: |
+          ../orca/bin/actions/self-test/before_install.sh
+          ../orca/bin/actions/before_install.sh
+      - name: Install
+        run: |
+          ../orca/bin/actions/self-test/install.sh
+          ../orca/bin/actions/install.sh
+      - name: Before script
+        run: ../orca/bin/actions/before_script.sh
+      - name: Script
+        run: |
+          ../orca/bin/actions/self-test/script.sh
+          ../orca/bin/actions/script.sh
+      - name: After script
+        run: ../orca/bin/actions/after_script.sh
+      - name: After success
+        if: ${{ success() }}
+        run: ../orca/bin/actions/after_success.sh
+      - name: After failure
+        if: ${{ failure() }}
+        run: ../orca/bin/actions/after_failure.sh
```

Comment:
The Travis CI configuration sends notifications to Slack (`#orca-status`) on status change (i.e., from passing to failing or vice versa). Should we try to add that functionality here, too? I suppose it makes sense for both CI platforms to report status as long as we officially support them both.

Reply:
I think this needs to be a follow-up issue. I could not find an official Slack Action, and while there are plenty of unofficial ones that I'm sure would work, they'd need to be vetted and approved.